.github/workflows/dart.yml

```diff
@@ -1,4 +1,4 @@
-# Created with package:mono_repo v6.5.7
+# Created with package:mono_repo v6.6.2
```

Comment:
We can also move away from using this mono_repo generated workflow file - in favor of some simple hand-written ones - if that's preferable.

Reply:
I think it's fine, I don't have any problems with it.

Reply:
https://github.com/google/protobuf.dart/pull/957 moves to using simple, hand-written workflow files per-package instead of a generated mono_repo one.